package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.25.0",
   "description": "node.js bindings for deltachat-core",
   "scripts": {
-    "rebuild": "node-gyp rebuild",
-    "submodule": "git submodule update --recursive --init",
-    "install": "node scripts/rebuild-all.js",
-    "generate-constants": "./scripts/generate-constants.js",
-    "test": "standard && npm run dependency-check && nyc node test/index.js",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "coverage-html-report": "rm -rf coverage/ && nyc report --reporter=html && opn coverage/index.html",
-    "dependency-check": "dependency-check package.json --missing --unused -i dependency-check -i nyc -i opn-cli -i standard -i napi-macros -i coveralls --entry index.js --entry test/index.js --entry scripts/generate-constants.js --entry scripts/rebuild-all.js"
+    "dependency-check": "dependency-check package.json --missing --unused -i dependency-check -i nyc -i opn-cli -i standard -i napi-macros -i coveralls --entry index.js --entry test/index.js --entry scripts/*.js",
+    "generate-constants": "./scripts/generate-constants.js",
+    "install": "node-gyp-build scripts/rebuild-core.js",
```

Comment:
This is important. At install time `node-gyp-build` will try to `require()` the module, if there are prebuilt binaries made, this step will be a no-op, if not it will:

* run `scripts/rebuild-core.js`
* run `node-gyp` to build the node bindings